### ISSUE TITLE
Research Request - Van Ness BRT Case Study

### ISSUE THREAD

**Issue by edasmalchi** [cal-itp/data-analyses#1212]
## Research Questions

1. How did the implementation of Van Ness BRT affect drivers on the corridor?

    - speeds
    - volumes/person-throughput

1. How did the implementation of Van Ness BRT affect transit riders on the corridor?

    - speeds
    - ridership/frequency/person-throughput

1. How effective is Van Ness at moving people _overall_ compared to other parts of the State Highway System? What would happen if we made those parts more like Van Ness?

## How will this research be used?
What will be achieved or informed? What is the intended impact?

## Stakeholders & End-Users
CT Sustainability, SFMTA, CT Traffic Ops
## Metrics
* any specific calculations
* you think will be needed

## Data sources
* **Cal-ITP data sources**:

- GTFS(RT) data warehouse
- Streetlight

* **External data sources**:

- Muni ridership

* **Remaining data source questions**:

## Deliverables
Notebook, dashboard, narrative, presentation details.

## Timeline of deliverables
Early Oct?

**Comment by edasmalchi** [comment]
Hey @natam1! @hunterowens , @hhmckay , Eric Sundquist and I talked this one over yesterday afternoon -- any chance your team could help out on the Streetlight side? Happy to chat more about the ask if helpful...

**Comment by natam1** [comment]
Yes! Of course! We could do some segment analyses pre and post BRT. I can write up some thoughts

**Comment by edasmalchi** [comment]
@natam1 @hhmckay have you had a chance to think about this one? I'll be circling back to it this week/next...

**Comment by natam1** [comment]
Yes! I wrote up some thoughts and ran a few initial studies in Streetlight and Replica but haven't had a chance to post process just yet

**Comment by KatrinaMKaiser** [comment]
TODO: sync on deliverable/report
